docs/ja/adc_driver.md

```diff
@@ -0,0 +1,155 @@
+# ADC ドライバ
+
+<!---
+  original document: 0.10.52:docs/adc_driver.md
+  git diff 0.10.52 HEAD -- docs/adc_driver.md | cat
+-->
+
+QMK は、サポートされる MCU のアナログ・デジタルコンバータ (ADC) を使って、特定のピンの電圧を計測することができます。これは、[ロータリーエンコーダ](ja/feature_encoders.md)とは対照的に、Bluetooth キーボードのバッテリーレベルインジケータや、可変抵抗器を使ったボリューム制御などの実装に役立ちます。
```

Comment:
ロータリーエンコーダが2値しかとらないのに対して、という意図かなぁと読んでました

Reply:
以下の二つは、as opposed to で対比する意味がありそうです。
* ロータリーエンコーダで、パソコン（ホスト）のボリュームをコントロールする。
* 可変抵抗器で、パソコン（ホスト）のボリュームをコントロールする。

しかし、「バッテリーレベルの表示（インジケータ）」は、上の二つと関係なさそうです。

なので、原文末尾の ', as opposed to a rotary encoder...' は、'or volume controls' から後にかかっているのではないでしょうか。

Reply:
それだとasの前にカンマがある意味がなく、
バッテリーレベル:ADCが使う
ボリュームコントロール:ADCを使う
QMKのロータリーエンコーダ:デジタル値
という前2つに対するロータリーエンコーダという対比にしているのだと思います

Reply:
うーん、、その解釈で訳出した文章は、なんか気持ち悪いです。

@ai03-2725 さん、助けてください。ここの訳はどうするのが正しいでしょうか？

Reply:
うーん、、その解釈で訳出した文章は、なんか気持ち悪いです。

@ai03-2725 さん、助けてください。ここの訳はどうするのが正しいでしょうか？

Reply:
雑な意見だと、最初のカンマは以下の記事でいうとこの2番で、2個目のカンマは以下の記事の3番ではないでしょうか。なので対比/並列になる要素はバッテリーインジケータとポテンショナルメータ、ポテンショナルメータとロータリーエンコーダだと思います。
https://www.rarejob.com/englishlab/other/20170509/

Reply:
そうするとmteiさんの訳でよさそう

Reply:
私の場合は日本語がボロボロなので力になれるかは分かりませんが、恐らくこのような翻訳で通じるはずです。

`
QMKは対応しているMCUのアナログ・デジタルコンバータ（ADC)を使用し、特定のピンの電圧を計測することができます。この機能はデジタル出力のロータリエンコーダなどではなく、アナログ計測が必要な可変抵抗器を使用したボリュームコントロールやBluetoothキーボードのバッテリー残量表示などの実装に役立ちます。
`

翻訳の際に元の文で抜けていた単語を追加しました。

`
QMK can leverage the Analog-to-Digital Converter (ADC) on supported MCUs to measure voltages on certain pins. This can be useful for implementing things [which depend on analog voltage readings] such as battery level indicators for Bluetooth keyboards, or volume controls using a potentiometer, as opposed to [digital output components such as] a rotary encoder.
`

Reply:
ご指摘ありがとうございます。
ai03さんの訳をそのまま頂きました。